glib/src/boxed.rs

```diff
@@ -354,7 +354,7 @@ macro_rules! glib_boxed_wrapper {
                 debug_assert_eq!(std::mem::size_of::<Self>(), std::mem::size_of::<$crate::ffi::gpointer>());
                 let value = &*(value as *const $crate::Value as *const $crate::gobject_ffi::GValue);
                 debug_assert!(!value.data[0].v_pointer.is_null());
-                <$name $(<$($generic),+>)?>::from_glib_ptr_borrow(&value.data[0].v_pointer as *const $crate::ffi::gpointer as *const *const $ffi_name)
+                <$name $(<$($generic),+>)?>::from_glib_ptr_borrow(&*(&value.data[0].v_pointer as *const $crate::ffi::gpointer as *const *mut $ffi_name))
```

Comment:
These changes make me a bit nervous. Does this create a temporary because of the `&value.data[0].v_pointer`, and then we return a reference to some long gone stack frame?

Reply:
We do the same in other places already so it should be fine